applications/portal/frontend/geppetto_setup.sh

```diff
@@ -0,0 +1,17 @@
+app=$(pwd)
```

Comment:
not needed for production, but useful for when we need to develop/fix stuff on geppetto for salk